src/pivotal-ui/components/svgeople/index.js

```diff
@@ -0,0 +1,103 @@
+var $ = require('jquery');
+require('@npmcorp/pui-css-forms').stepper;
+
+var SVGeople = function(el, amountOfPeopleFiles, shownCount, root) {
+
+  this.el = $(el);
+  this.root = root || this.el.data("asset-root") || "/static/images/";
+  amountOfPeopleFiles = amountOfPeopleFiles || this.el.data("amount-people-files") || 10;
+  this.shownCount = shownCount || this.el.data("starting-count") || 1;
+
+  this.peopleFiles = [];
+  for (var i = 0; i < amountOfPeopleFiles; i++) {
+    this.peopleFiles.push(this.root + "people_p" + i + ".svg");
+  }
+
+  this.people = [];
+};
+
+SVGeople.prototype.addPerson = function(idx) {
+  var id = "#svgerson_" + idx;
```

Comment:
What about getting the nth that matches a selector and removing that? Working by actual index rather than a stored ID?